lib/rss.js

```diff
@@ -0,0 +1,192 @@
+const RSS = require('rss')
+const generateDiff = require('deep-diff').diff
+const fs = require('fs')
+
+function createLeadersHash(array) {
+  return array.reduce((hash, elem) => {
+    hash[elem.id] = elem
+    return hash
+  }, {})
+}
+
+function createOrgHash(array) {
+  let rank = 0
+  return array.reduce((hash, elem) => {
+    rank++
+    elem.rank = rank
+    elem.leaders = createLeadersHash(elem.leaders)
+    hash[elem.slug] = elem
+    return hash
+  }, {})
+}
+
+let current = {}
+try {
+  current = JSON.parse(
+    fs.readFileSync(`${__dirname}/../out/feed_items.json`).toString()
```

Comment:
We might want to fetch this like `data.json` or the file will not be available on every build.

Reply:
Alright done now in `gather.js` file.